framework/entity/src/main/java/org/apache/ofbiz/entity/util/Queryable.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.ofbiz.entity.util;
+
+import org.apache.ofbiz.entity.Delegator;
+
+/**
+ * Interface used for the <code>EntityQuery</code> to set the delegator to yse for the query.
+ * Allows easyer and shorter query instructions, like calling an <code>EntityQuery</code> with a dispatcher.
+ */
+public interface Queryable {
+
+    /**
+     * Gets the GenericEntityDelegator associated with this dispatcher
```

Comment:
The Javadoc is not correct as it mentions GenericEntityDelegator rather than Delegator and it mentions "this dispatcher" which is out of context in this interface.